gatsby-site/src/components/Contact/contact.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.form-container {
+    width: 100%;
+    text-align: center;
+    padding: 40px;
+}
+
+.left-image {
+    width: 50%;
+    margin: 20px;
+}
+
+.right-form {
+    width: 40%;
+    word-break: keep-all;
+}
+
+.right-form, .left-image{
+    display: inline-block;
+    padding: 50px;
+    align-self: center;
+}
+
+.submitbtn {
+    float: right;
+    width: 40%;
+    padding: 0;
+    font-family: var(--colarkfont);
+    margin-top: 25px;
+    background-color: #FF6969;
```

Comment:
I think if we are going to use css vars then we should use them in all of the places that use repeated hex codes.

Reply:
I'm taking care of it in my global-css pr, coming soon =)